ground/src/main/proto/com/google/ground/job.proto

```diff
@@ -19,11 +19,189 @@ syntax = "proto3";
+    message DateTimeQuestion {
+        // Allowed values for the `type` field.
+        enum Type {
+            UNSPECIFIED_DATE_TIME_QUESTION_TYPE = 0;
+            DATE_ONLY = 1;
+            TIME_ONLY = 2;
+            BOTH_DATE_AND_TIME = 3;
+        }
+        // The type of data/time question.
+        Type type = 1;
+    }
+
+    // A question whose response is manually selected from a list of options.
+    message MultipleChoiceQuestion {
+        // Valid values for the `type` field.
+        enum Type {
+            UNSPECIFIED_MULTIPLE_CHOICE_QUESTION_TYPE = 0;
+            // Allow only one valid to be selected.
+            SELECT_ONE = 1;
+            // Allow one or more values to be selected.
+            SELECT_MULTIPLE = 2;
+        }
+        // The type of multiple choice question.
+        Type type = 1;
+        // The list of allowed multiple choice options.
+        repeated Option options = 2;
+
+        // A single selectable multiple choice option.
+        message Option {
+            // UUID of the option.
+            string id = 1;
+            // The index of the option when shown as a list.
+            int32 index = 2;
+            // The user-facing label of the option.
+            string label = 3;
+            // When true, the user may add qualifying text to the option. If
+            // label is omitted, clients should apply a default label
+            // "Other (specify)" or equivalent.
+            bool other_option = 4;
+        }
+    }
+
+    // A task in which the user must manually draw a geometry (point,
+    // polygon, etc.) on the map.
+    message DrawGeometry {
+        // Possible values for the `allowed_methods` field.
+        enum Method {
+            UNSPECIFIED_DRAW_GEOMETRY_METHOD = 0;
+            DROP_PIN = 1;
+            DRAW_AREA = 2;
+        }
+        // One or more methods which the user may use to complete the geometry
+        // task. If multiple methods are specified, data collectors are prompted
+        // which method should be used.
+        repeated Method allowed_methods = 1;
+    }
+
+    // A task in which the user must capture their current device location.
+    message CaptureLocation {
+        // Placeholder message. Fields may be added here in the future.
+    }
+
+    // A task in which the user must take a photo.
+    message TakePhoto {
+        // Placeholder message. Fields may be added here in the future.
+    }
+
+    // Defines a single condition used to determine whether a task should be
+    // shown.
+    message Condition {
+        // A condition is defined by one of the following constraints:
+        oneof condition_type {
+            // A condition which passes if any of the specified values are
+            //selected.
+            MultipleChoiceSelection multiple_choice = 1;
+        }
+    }
+
+    // A condition which passed if any of the specified values are selected.
+    message MultipleChoiceSelection {
```

Comment:
This feels like it should be pluralized, since the condition accounts for multiple options, but the style guide probably would discourage non-repeated fields to be plural. Instead, maybe `MultipleChoiceCondition` to make clear the association between this message type and the Condition one?

Or consider nesting this proto.

https://cloud.google.com/apis/design/naming_convention#repeated_field_names

Reply:
Since this message type in nested, the selection options are already plural and are namespaced as`Task.Condition.MultipleChoiceSelection.option_ids`. In that case do you think it's clear?

Reply:
Oh gosh! I'm sorry, I couldn't read the proto nesting... Looks good!